package/kernel/linux/modules/netsupport.mk

```diff
@@ -1017,7 +1017,8 @@ define KernelPackage/sched
 	CONFIG_NET_EMATCH_META \
 	CONFIG_NET_EMATCH_TEXT
   FILES:=$(SCHED_FILES_EXTRA)
-  AUTOLOAD:=$(call AutoLoad,73, $(SCHED_MODULES_EXTRA))
+  FILES+=$(LINUX_DIR)/net/sched/sch_dsmark.ko@lt6.1
+  AUTOLOAD:=$(call AutoLoad,73, $(SCHED_MODULES_EXTRA) sch_dsmark@lt6.1)
```

Comment:
Just remove this unconditionally 5.15.150 also removes `sch_dsmark`

Reply:
Are you saying drop the `@lt6.1` from both new lines?

Reply:
Just remove it from the package like this: https://github.com/openwrt/openwrt/pull/14777/commits/ac4985b91621ffe5bc80fb4bdd3af377a51f1e36

Reply:
Done